test/getCommands.spec.js

```diff
@@ -0,0 +1,50 @@
+const path = require('path');
+const Module = require('module');
+const expect = require('chai').expect;
+const getCommands = require('../src/getCommands');
+const findPlugins = require('../src/findPlugins');
+const mockRequire = require('mock-require');
+const mockFs = require('mock-fs');
+
+const commands = getCommands();
```

Comment:
Can we call `getCommands()` in every suite separately instead of having global reference?